src/views/taskList.html

```diff
@@ -0,0 +1,179 @@
+<main id="content" role="main">
+
+    {{> common/betaBanner}}
+
+    <div class="grid-row">
+        <div class="column-two-thirds column-minimum">
+
+            <form method="post" action="/v7/check/check-answers" name="tasklist">
+
+                <h1 class="heading-large">
+    <span class="font-medium" style="color:#6f777b">Apply for a standard rules waste permit:</span><br>
+    Anaerobic digestion facility including use of the resultant biogas (waste recovery operation)<br>
+    <span class="heading-medium">SR2012 No 12 &nbsp;
+      <a class="font-small" href="/v7/selectpermit/permit-category2">select a different permit</a>
+    </span>
+    </h1>
+
+                <ol class="task-list">
+
+                    <!-- Before you apply ############################################################### -->
+                    <li>
+                        <h2 class="task-list-section">
+              <span class="task-list-section-number">1. </span> Before you apply
+            </h2>
+                        <ul class="task-list-items">
+                            <li class="task-list-item " id="check-permit-cost-and-time">
+
+                                <a class="task-name" href="/v7/selectpermit/cost-and-time" aria-describedby="cost-and-time-completed">
+                  Check permit cost and time
+                </a>
+
+                                <input type="hidden" name="complete[]" value="Check permit cost and time">
```

Comment:
Agree that IDs are needed to make testing easier. We definitely don't want to be traversing the DOM!

I'm not aware of any automated method of adding static IDs, I have always added them manually, keeping them unique and human readable by having some kind of naming convention, e.g. id="PAGE_NAME_ELEMENT_NAME_ELEMENT_INDEX" 
e.g. CUSTOMERS_CUSOMTER_LIST_ITEM_0.

Rather than use HTML IDs though, another option is to use a custom data-* attribute
e.g. data_test="......" instead of id="....."

There are two advantages of this:
1) We avoid clashes with CSS, where the CSS writer wants to use IDs to help them style the page
2) I have got a Gulp task that removes all of these 'data_test' attributes as part of a release build task so that they don't make it into production (making our site smaller to download).

 Happy to use IDs though, just a suggestion!

Either way I don't think that there is much value in adding in anything now as the templates I have added are only placeholders that I have copied out of the design prototype and therefore subject to change. Happy to address this once we start working on the proper pages.